### PR TITLE
Headers update for consequence network and operator page

### PR DESCRIPTION
**Pull request by alvak1**
Access pages using the below path

`/create-consequence-network`
`/create-consequence-operator`

### CODE REVIEW

File: site/pages/__snapshots__/create-consequence-network.test.tsx.snap [hunk 1]
```diff
@@ -273,7 +267,7 @@ exports[`pages > CreateConsequenceNetwork > should render correctly with inputs
                 className="govuk-label govuk-label--l"
                 htmlFor="disruption-severity-input"
               >
-                What is the severity of the disruption?
+                Disruption Severity
```
Comment (alainajaffer) [inline site/pages/__snapshots__/create-consequence-network.test.tsx.snap:270]:
```suggestion
                Disruption severity
```

File: site/pages/__snapshots__/create-consequence-network.test.tsx.snap [hunk 2]
```diff
@@ -793,7 +701,7 @@ exports[`pages > CreateConsequenceNetwork > should render correctly with no inpu
                 <span
                   className="govuk-heading-l govuk-!-margin-bottom-0"
                 >
-                  Would you like to remove this from journey planners?
+                  Remove from journey planners?
```
Comment (alainajaffer) [inline site/pages/__snapshots__/create-consequence-network.test.tsx.snap:704]:
```suggestion
                  Remove from journey planners
```

File: site/pages/__snapshots__/create-consequence-operator.test.tsx.snap [hunk 3]
```diff
@@ -326,7 +320,7 @@ exports[`pages > CreateConsequenceOperator > should render correctly with inputs
                 className="govuk-label govuk-label--l"
                 htmlFor="disruption-severity-input"
               >
-                What is the severity of the disruption?
+                Disruption Severity
```
Comment (alainajaffer) [inline site/pages/__snapshots__/create-consequence-operator.test.tsx.snap:323]:
```suggestion
                Disruption severity
```

File: site/pages/__snapshots__/create-consequence-operator.test.tsx.snap [hunk 4]
```diff
@@ -998,7 +900,7 @@ exports[`pages > CreateConsequenceOperator > should render correctly with no inp
...
```
Comment (alainajaffer) [inline site/pages/__snapshots__/create-consequence-operator.test.tsx.snap:903]:
```suggestion
                Disruption severity
```

File: site/pages/create-consequence-network.page.tsx [hunk 5]
```diff
@@ -136,40 +135,16 @@ const CreateConsequenceNetwork = ({
 
                         <Select<ConsequenceNetworkPageInputs>
                             inputName="disruptionSeverity"
-                            display="What is the severity of the disruption?"
+                            display="Disruption Severity"
```
Comment (alainajaffer) [inline site/pages/create-consequence-network.page.tsx:138]:
```suggestion
                            display="Disruption severity"
```

File: site/pages/__snapshots__/create-consequence-network.test.tsx.snap (hunk 1 above)
Comment (alvak1) [reply site/pages/__snapshots__/create-consequence-network.test.tsx.snap:270]: done

File: site/pages/__snapshots__/create-consequence-operator.test.tsx.snap (hunk 3 above)
Comment (alvak1) [reply site/pages/__snapshots__/create-consequence-operator.test.tsx.snap:323]: done

File: site/pages/__snapshots__/create-consequence-operator.test.tsx.snap [hunk 6]
```diff
@@ -899,7 +807,7 @@ exports[`pages > CreateConsequenceOperator > should render correctly with no inp
                 <span
                   className="govuk-heading-l govuk-!-margin-bottom-0"
                 >
-                  Would you like to remove this from journey planners?
+                  Remove from journey planners?
```
[the remark this answers is not in the document]
Comment (alvak1) [reply site/pages/__snapshots__/create-consequence-operator.test.tsx.snap:810]: done

File: site/pages/__snapshots__/create-consequence-operator.test.tsx.snap (hunk 4 above)
Comment (alvak1) [reply site/pages/__snapshots__/create-consequence-operator.test.tsx.snap:903]: done

**Review by alainajaffer — DISMISSED** [review]

**Review by lj-kpmg — APPROVED** [review]

**Pull request merged**